test/Microsoft.AspNetCore.Server.KestrelTests/ListenerPrimaryTests.cs

```diff
@@ -189,6 +189,12 @@ public class ListenerPrimaryTests
 
                 await kestrelThreadPrimary.PostAsync(_ => pipe.Dispose(), null);
 
+                // Wait up to 10 seconds for error to be logged
+                for (var i = 0; i < 10 && primaryTrace.Logger.TotalErrorsLogged == 0; i++)
```

Comment:
I know you just copied this from #1230, but this only waits up to 1 second not 10 seconds. I would update `< 10` to `< 300` for a 30 second timeout to match `AssertResponseEventually`.

Reply:
Derp, didn't see this until now, but I made a similar comment in the other PR.